envoy-control-core/src/test/kotlin/pl/allegro/tech/servicemesh/envoycontrol/groups/NodeMetadataTest.kt

```diff
@@ -42,18 +42,51 @@ class NodeMetadataTest {
 
         // expects
         val exception = assertThrows<NodeMetadataValidationException> { proto.toIncomingEndpoint() }
-        assertThat(exception.status.description).isEqualTo("Precisely one of 'path' and 'pathPrefix' field is allowed")
+        assertThat(exception.status.description).isEqualTo("Precisely one of 'path', 'pathPrefix' or 'pathRegex' field is allowed")
         assertThat(exception.status.code).isEqualTo(Status.Code.INVALID_ARGUMENT)
     }
 
     @Test
-    fun `should reject endpoint with no path or pathPrefix defined`() {
+    fun `should reject endpoint with both path and pathRegex defined`() {
```

Comment:
Maybe we can use `ParametrizedTest` to cover all cases?

Reply:
done